lib/eventDrops.js

```diff
@@ -2,7 +2,7 @@ import d3 from 'd3';
 
 import configurable from 'configurable.js';
 import defaultConfig from './config';
-import drawer from './drawer';
+import drawer from './drawer/index.js';
```

Comment:
I'm not very confortable with this change. Is it compulsory for jspm? Isn't there any module to counter that?

If not, write at least a comment to let the `index.js` file here. Otherwise, another developer would probably remove it in a near future. :)